packages/mds-audit/service.ts

```diff
@@ -149,6 +161,17 @@ export function withGpsProperty<T extends TelemetryData>({
   return { ...props, gps: { lat, lng, speed, heading, accuracy, hdop, altitude, satellites } }
 }
 
+export async function getVehicle(provider_id: UUID, vehicle_id: string) {
+  const device = await readDeviceByVehicleId(provider_id, vehicle_id)
+  const deviceStatus = (await cache.readDeviceStatus(device ? device.device_id : '')) as (VehicleEvent & Device)
```

Comment:
What does `readDeviceStatus` do when passed no device_id (`''`)? Should it be inside an `if (device) {}`?

Reply:
Yeah, I kind of went back and forth on that. I had it set up to pass through and eventually hit the cache and return null. I updated it to add a null check on `device` here though and avoid the cache hit.